third_party/freedreno/includes/instr-a3xx.h

```diff
@@ -748,6 +757,10 @@ typedef union PACKED {
 	};
 } instr_t;
 
+#if defined(_MSC_VER)
```

Comment:
could you also add `// GOOGLE: <reason for modification>` here? so that it would be easier for future integration. thanks :)